cmd/notary-signer/passwordwrap/interface.go

```diff
@@ -0,0 +1,19 @@
+package passwordwrap
```

Comment:
Thinking all of this should just be under the `passphrase` module. Thoughts @riyazdf @cyli @ecordell ??